config.mk

```diff
@@ -108,15 +108,15 @@ makejobs=-j2
 # hard drive space.
 pass2_languages=c,c++,objc,obj-c++
 
-# Download protocol (https|ftp)
+# Download protocol (http|ftp)
 # Specify here the protocol you want to use for downloading the packages.
-download_protocol=https
+download_protocol=http
```

Comment:
Maybe we may add `http|https|ftp` protocoles?

Reply:
it should be `http` only because the download tool redirects to `https`.
if you use `https` the `http` only URL doesn't work.